website/events/payables.py

```diff
@@ -8,7 +8,11 @@
 class EventRegistrationPayable(Payable):
     @property
     def payment_amount(self):
-        return self.model.event.price
+        return (
+            self.model.event.price
+            if not self.model.special_price
+            else self.model.special_price
+        )
```

Comment:
I feel like this logic should be part of the event registration, give it a property that returns the right price for that specific registration so that you can use it in all places and not have to repeat the logic.